CHANGES.rst

```diff
@@ -4,6 +4,11 @@ Change Log
 git master
 ----------
 
+New features
+''''''''''''
+
+* Added ``min_reported_time`` configurable.
```

Comment:
You will need to shortly explain what this does.

Reply:
done